R/pkg_ref_cache_bug_reports_host.R

```diff
@@ -12,5 +12,5 @@ pkg_ref_cache.bug_reports_host <- function(x, ...) {
 #' @importFrom urltools host_extract domain
```

Comment:
Let's remove the `host_extract` function from our namespace now that we're no longer using it while we're making these edits.

```suggestion
#' @importFrom urltools domain
```